tests/tests.py

```diff
@@ -247,38 +239,6 @@ def test_save_field_to_database(self):
             transform=phone_transform,
         )
 
-    def test_create_invalid_number(self):
```

Comment:
Rather than removing these tests, please alter them to assert the new behavior and side effects.

Reply:
Did you have a look at the removed tests? They were asserting the exceptions, which I was explicitly emphasizing that it SHOULD NOT raise.

Reply:
Understood. 

If we're going to change the existing behavior, we should now assert the expected behavior when saving an invalid number. 

When saving an invalid number, the side effect is ... ?

The test should answer this question.
So if we behavior change the behavior from raise an exception